shellwhat/parsers.py

```diff
@@ -7,7 +7,10 @@
 
 class OshNode(AstNode):
     def get_text(self, full_text=None):
-        return self.text
+        if self.text:
+            return self.text
+        else:
+            raise RuntimeError("The parser wasn't able to match text to this node")
```

Comment:
Guessing RunTimeError will be caught by the SCT runner, and won't crash the backend, if it RunTimeErrors on solution code?

Reply:
The two places where it's used (and can be caught) are:
- [has_equal_ast](https://github.com/datacamp/protowhat/blob/7b6002d32c5104a31de3e43c06b994aac6b60d19/protowhat/checks/check_funcs.py#L294)
- [has_code](https://github.com/datacamp/protowhat/blob/7b6002d32c5104a31de3e43c06b994aac6b60d19/protowhat/checks/check_funcs.py#L216)

But I was't sure whether this was the best exception type. Which one would you use? 
There's already [ParseError](https://github.com/datacamp/protowhat/blob/7b6002d32c5104a31de3e43c06b994aac6b60d19/protowhat/utils_ast.py#L75), would you introduce another custom error type? (AstError?)